README.md

```diff
@@ -1,57 +1,60 @@
-- *v0.0.13* 
+- *v0.0.13*
   - Updated IBM Cloud icon
-- *v0.0.12* 
+- *v0.0.12*
   - Added support for `bx dev shell` command
   - Removed support for the `sdkgen` cli
   - Improved cli version detection with minimum versioning and forced upgrade paths to support new cli features
   - Added support for "caller" arguments for the `dev` cli
-- *v0.0.11* 
+- *v0.0.11*
   - Added support for Kubernetes/Helm deployment
   - Added support for `bx dev console`
-- *v0.0.10* 
-  - Updated badges in README for VS Code marketplace compliance.  
+- *v0.0.10*
+  - Updated badges in README for VS Code marketplace compliance.
   - Fixed "killed terminal" bug in login/logout commands
 - *v0.0.9* - Updated usage/getting started in README
 - *v0.0.8* - first public release
 
 ## Usage
 
-Easily invoke commands from the IBM Bluemix Developer CLI from directly inside of the VS Code editor:
+Easily invoke commands from the IBM Cloud Developer CLI from directly inside of the VS Code editor:
 
-- Open the VS Code command pallette (`F1` or `CMD-Shift-P`)
-- Use the `bx login` command to log in to Bluemix (using your Bluemix credentials)
+- Open the VS Code command palette (`F1` or `CMD-Shift-P`)
+- Use the `ibmcloud login` command to log in to IBM Cloud (using your IBM Cloud credentials)
 
-### Using `bx dev` workflow: 
-Video demonstrating `bx dev` workflow: [IBM Developer Extension for VS Code
+### Using `ibmcloud dev` workflow:
+Video demonstrating `ibmcloud dev` workflow: [IBM Developer Extension for VS Code
 ](https://youtu.be/JmrLaLRRQSk)
 
 Steps to get started:
 - Create a project using one of the two methods below:
-    - `bx dev create` using the Bluemix Developer CLI (outside of VS Code)
-    - [Bluemix Web console](https://console.ng.bluemix.net/developer/getting-started/)
+    - `ibmcloud dev create` using the IBM Cloud Developer CLI (outside of VS Code)
+    - [IBM Cloud Developer Documentation](https://cloud.ibm.com/docs?tab=develop)
 - Open the *project’s folder* locally in the VS Code editor
-- Use the `bx dev build` command to build the app into a Docker image
-- Use the `bx dev debug` command to run the app in local Docker for development
-- Use the `bx dev run` command to run the app in local Docker in release mode
-- Use the `bx dev deploy` command to deploy the app (in the Docker container) to Bluemix
-- Use the `bx dev console` command to open your project on Bluemix in a web browser
-
-### Using CloudFoundry workflow: 
+- Use the `ibmcloud dev build` command to build the app into a Docker image
+- Use the `ibmcloud dev debug` command to run the app in local Docker for development
+- Use the `ibmcloud dev run` command to run the app in local Docker in release mode
+- Use the `ibmcloud dev deploy` command to deploy the app (in the Docker container) to IBM Cloud
+- Use the `ibmcloud dev view` command to open your project on IBM Cloud in a web browser
```

Comment:
A better description would be `to open your deployed app`

Reply:
Will do